ruby/private/bundle/create_bundle_build_file.rb

```diff
@@ -252,12 +252,23 @@ def register_gem(spec, template_out, bundle_lib_paths, bundle_binaries)
     # Do not register local gems
     return if spec.source.path?
 
-    gem_path = GEM_PATH[ruby_version, spec.name, spec.version]
-    spec_path = SPEC_PATH[ruby_version, spec.name, spec.version]
-    base_dir = "lib/ruby/#{ruby_version}"
+    if spec.source.is_a?(Bundler::Source::Git)
+      stub = spec.source.specs.find { |s| s.name == spec.name }.stub
+      base_dir = "lib/ruby/#{ruby_version}"
```

Comment:
Define `base_dir` before if since it's the same for both forks.

Reply:
fix it 👍